{"metadata":{"externalID":"s:12AsyncObjects11ContinuableP7FailureQa","modules":[{"name":"AsyncObjects"}],"required":true,"symbolKind":"associatedtype","title":"Failure","fragments":[{"kind":"keyword","text":"associatedtype"},{"kind":"text","text":" "},{"kind":"identifier","text":"Failure"},{"kind":"text","text":" : "},{"kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP","text":"Error"}],"role":"symbol","roleHeading":"Associated Type"},"primaryContentSections":[{"declarations":[{"tokens":[{"text":"associatedtype","kind":"keyword"},{"text":" ","kind":"text"},{"text":"Failure","kind":"identifier"},{"text":" : ","kind":"text"},{"text":"Error","preciseIdentifier":"s:s5ErrorP","kind":"typeIdentifier"}],"languages":["swift"],"platforms":["Linux"]}],"kind":"declarations"}],"kind":"symbol","schemaVersion":{"patch":0,"minor":3,"major":0},"abstract":[{"type":"text","text":"The type of error to resume the continuation with in case of failure."}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/asyncobjects\/continuable\/failure"]}],"sections":[],"hierarchy":{"paths":[["doc:\/\/AsyncObjects\/documentation\/AsyncObjects","doc:\/\/AsyncObjects\/documentation\/AsyncObjects\/Continuable"]]},"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/AsyncObjects\/documentation\/AsyncObjects\/Continuable\/Failure"},"references":{"doc://AsyncObjects/documentation/AsyncObjects/Continuable/Failure":{"title":"Failure","kind":"symbol","role":"symbol","type":"topic","abstract":[{"type":"text","text":"The type of error to resume the continuation with in case of failure."}],"fragments":[{"text":"associatedtype","kind":"keyword"},{"text":" ","kind":"text"},{"text":"Failure","kind":"identifier"},{"text":" : ","kind":"text"},{"text":"Error","preciseIdentifier":"s:s5ErrorP","kind":"typeIdentifier"}],"url":"\/documentation\/asyncobjects\/continuable\/failure","identifier":"doc:\/\/AsyncObjects\/documentation\/AsyncObjects\/Continuable\/Failure","required":true},"doc://AsyncObjects/documentation/AsyncObjects":{"title":"AsyncObjects","kind":"symbol","role":"collection","type":"topic","abstract":[{"type":"text","text":"Several synchronization primitives and task synchronization mechanisms introduced to aid in modern swift concurrency."}],"url":"\/documentation\/asyncobjects","identifier":"doc:\/\/AsyncObjects\/documentation\/AsyncObjects"},"doc://AsyncObjects/documentation/AsyncObjects/Continuable":{"title":"Continuable","kind":"symbol","role":"symbol","type":"topic","abstract":[{"type":"text","text":"A type that allows to interface between synchronous and asynchronous code,"},{"type":"text","text":" "},{"type":"text","text":"by representing task state and allowing task resuming with some value or error."}],"navigatorTitle":[{"text":"Continuable","kind":"identifier"}],"fragments":[{"text":"protocol","kind":"keyword"},{"text":" ","kind":"text"},{"text":"Continuable","kind":"identifier"}],"url":"\/documentation\/asyncobjects\/continuable","identifier":"doc:\/\/AsyncObjects\/documentation\/AsyncObjects\/Continuable"}}}